src/app.py

```diff
@@ -156,6 +160,20 @@ def get_streak(event):
     print(streak)
     return int(streak)
 
+def get_score(event):
+    #get score value in text list
+    blocks = event.get("event").get("blocks")
+    elements = blocks[0].get("elements")
+    elements = elements[0].get("elements")
+    score_text = [elem for elem in elements if ('text' in elem and "/5" in elem.get("text"))][0]
```

Comment:
What is `/5`?

Reply:
We are grabbing the element that contains the score. In the format of the wafflegame
```
#waffle727 4/5
:large_green_square::large_green_square::large_green_square::large_green_square::large_green_square:
:large_green_square::star::large_green_square::star::large_green_square:
:large_green_square::large_green_square::large_green_square::large_green_square::large_green_square:
:large_green_square::star::large_green_square::star::large_green_square:
:large_green_square::large_green
```
The element in question will be 
#waffle727 4/5
So the score will always been in the element text block with "/5"